packages/app/components/auth.component.tsx

```diff
@@ -52,8 +52,14 @@ export const Auth: React.FC<AuthProps> = ({ navigation }) => {
               subtitle={currentLanguageName}
               accessoryLeft={() => (
                 <TopNavigationAction
+                  accessibilityHint={translate(
+                    'auth.a11y_navigate_to_change_language',
+                    {
+                      defaultValue: 'Navigerar till byt språk skärmen',
```

Comment:
```suggestion
                      defaultValue: 'Navigerar till vyn för att byta språk',
```

Maybe this makes more sense?